app/services/reactivate_account_session.rb

```diff
@@ -27,23 +27,20 @@ def suspend
   # Stores PII as a string in the session
   # @param [Pii::Attributes]
   def store_decrypted_pii(pii)
-    reactivate_account_session[:personal_key] = true
     reactivate_account_session[:validated_personal_key] = true
     pii_json = pii.to_json
-    reactivate_account_session[:pii] = pii_json
     Pii::Cacher.new(@user, session).save_decrypted_pii_json(pii_json)
     nil
   end
 
   def validated_personal_key?
-    reactivate_account_session[:personal_key]
+    reactivate_account_session[:validated_personal_key]
```

Comment:
on `main` should this have read from both old and new key?

Reply:
I was having trouble wrapping my head around that when I was working on it.  In this case the original value should 100% be there for the first deploy, and same for the new one in the deploy after that one, so it didn't seem necessary?

Reply:
I guess because the approach was to send both new + old at the same time and not a rename... you're right this is fine as-is